pkg/network/config/usm_config.go

```diff
@@ -191,9 +191,9 @@ func NewUSMConfig(cfg model.Config) *USMConfig {
 		MaxKafkaStatsBuffered: cfg.GetInt(sysconfig.FullKeyPath(smNS, "max_kafka_stats_buffered")),
 
 		// Postgres Protocol Configuration
-		EnablePostgresMonitoring:   cfg.GetBool(sysconfig.FullKeyPath(smNS, "enable_postgres_monitoring")),
-		MaxPostgresStatsBuffered:   cfg.GetInt(sysconfig.FullKeyPath(smNS, "max_postgres_stats_buffered")),
-		MaxPostgresTelemetryBuffer: cfg.GetInt(sysconfig.FullKeyPath(smNS, "max_postgres_telemetry_buffer")),
+		EnablePostgresMonitoring:   cfg.GetBool(sysconfig.FullKeyPath(smNS, "postgres", "enabled")),
+		MaxPostgresStatsBuffered:   cfg.GetInt(sysconfig.FullKeyPath(smNS, "postgres", "max_stats_buffered")),
+		MaxPostgresTelemetryBuffer: cfg.GetInt(sysconfig.FullKeyPath(smNS, "postgres", "max_telemetry_buffer")),
```

Comment:
What about legacy configuration ? Are those setting already used by customer (documented in some way) ?

We don't want configuration to break upon upgrading the Agent (since we are only bumping the minor and not the major when releasing a new Agent). Usually for such change we add a small piece of logic that coalesce old and new setting with precedence for the old one.

Reply:
No one is using it yet, that's why we feel comfortable breaking it